composer.json

```diff
@@ -24,6 +24,7 @@
         "nyholm/psr7": "^1.2",
         "php-coveralls/php-coveralls": "^2.1",
         "php-http/guzzle6-adapter": "^2.0",
+        "phpstan/phpstan": "^0.12.18",
```

Comment:
might want to change this to ``"^0.12"``

Reply:
Yeah composer chose this constraint with `composer req phpstan/phpstan --dev` :yum:

Reply:
yes composer behaves a bit different when packages have a version < 1
changing it to "^0.12" will at least ensure we're up to date with the patches

Reply:
Changed it.

Although: Isn't `^0.12.18` the same as `>=0.12.18, <0.13`?

Reply:
yes you're right :|
i was confused and thought it would stick to the patches, but sticks to the minor for pre 1.0 releases...